ledger/heavy/exporter/record_exporter_server_test.go

```diff
@@ -585,6 +585,18 @@ func TestRecordServer_Export_Composite(t *testing.T) {
 		require.Equal(t, secondRec, resRecord.Record)
 	})
 
+	t.Run("context.Canceled error", func(t *testing.T) {
+		streamMock := &streamMock{checker: func(i *Record) error {
+			return context.Canceled
+		}}
+
+		err := recordServer.Export(&GetRecords{
+			PulseNumber:  firstPN,
+			RecordNumber: 1,
+			Count:        1,
+		}, streamMock)
+		require.Error(t, err)
```

Comment:
If we want to test a specific behaviour, we should assert a specific error. This will assert a general error that could be caused by an unexpected behaviour.